.trivyignore

```diff
@@ -1,5 +1,7 @@
 # Jenkins CVEs
 CVE-2016-1000027
+CVE-2024-25710
```

Comment:
I think these CVEs can be removed (perhaps on another PR to help reduce complexity)